app/assets/javascripts/application/suggest_user.js

```diff
@@ -0,0 +1,24 @@
+function suggestUser() {
+    var url = "/search/suggest_leader";
+    var params = "query=" + document.getElementsByName('group_leader_name')[0].value;
+    http = new XMLHttpRequest();
+    http.open("GET", url + "?" + params, true);
+    http.onreadystatechange = function () {//Call a function when the state changes.
+        if (http.readyState == 4 && http.status == 200) {
+            addResultToSuggestions(http.responseText);
+        }
+    }
+    http.send(null);
+}
+
+function addResultToSuggestions(result) {
+    console.log(result);
+    // $(".result").html();
+    suggestions = document.getElementById("user_suggestions");
+    suggestions.innerHTML = '';
+    suggestions.insertAdjacentHTML('beforeend', result);
+}
+
+function selectUser(user_id) {
+    $("#selected_user_id")[0].value = user_id;
```

Comment:
Let's try to avoid using jQuery.

Reply:
Removed it. I think no more jQuery left in this file.